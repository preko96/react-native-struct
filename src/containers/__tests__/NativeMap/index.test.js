import React from 'react'
import { NativeMap } from '../../NativeMap/'

it('renders without crashing', () => {
	expect(shallow(<NativeMap />)).toMatchSnapshot()
})

it('renders with found location', () => {
	expect(shallow(<NativeMap latitude={0} longitude={0}/>)).toMatchSnapshot()
})