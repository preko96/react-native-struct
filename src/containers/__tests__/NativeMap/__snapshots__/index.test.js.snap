// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with found location 1`] = `
<Component
  style={
    Array [
      Object {
        "alignItems": "center",
        "bottom": 0,
        "justifyContent": "center",
        "left": 0,
        "position": "relative",
        "right": 0,
        "top": 0,
      },
      Object {},
      Object {
        "height": undefined,
      },
    ]
  }
>
  <MapView
    pitchEnabled={true}
    region={
      Object {
        "latitude": 0,
        "latitudeDelta": 0.005,
        "longitude": 0,
        "longitudeDelta": 0.005,
      }
    }
    rotateEnabled={true}
    scrollEnabled={true}
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
    zoomEnabled={true}
  >
    <AnimatedComponent
      coordinate={
        Object {
          "latitude": 0,
          "latitudeDelta": 0.005,
          "longitude": 0,
          "longitudeDelta": 0.005,
          "valid": true,
        }
      }
    />
  </MapView>
</Component>
`;

exports[`renders without crashing 1`] = `
<Component
  style={
    Array [
      Object {
        "alignItems": "center",
        "bottom": 0,
        "justifyContent": "center",
        "left": 0,
        "position": "relative",
        "right": 0,
        "top": 0,
      },
      Object {},
      Object {
        "height": undefined,
      },
    ]
  }
>
  <MapView
    pitchEnabled={false}
    region={
      Object {
        "latitude": 54.3988402,
        "latitudeDelta": 10,
        "longitude": -4.5765776,
        "longitudeDelta": 10,
      }
    }
    rotateEnabled={false}
    scrollEnabled={false}
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
    zoomEnabled={false}
  />
  <Component
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(255, 255, 255, 0.8)",
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Styled(H1)
      style={
        Object {
          "color": "grey",
        }
      }
    >
      Location not found
    </Styled(H1)>
  </Component>
</Component>
`;
